src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example1.txt

```diff
@@ -24,6 +24,6 @@ while (counter < 10)          // violation, single-line statements not allowed w
     ++count;
 while (value.incrementValue() < 5); // violation, empty loop body not allowed
 switch (num) {
-  case 1: counter++; break;         // OK
+  case 1: counter++; break;         // OK, Exception Case
```

Comment:
`// OK, 'break' is not counted to allow compact view

Reply:
Please add attention note to description https://checkstyle.org/checks/blocks/needbraces.html#Description

Reply:
@romani  // Ok, break would be good wording

Reply:
Attention: break in case blocks is not counted to allow compact view

Reply:
> Please add attention note to description https://checkstyle.org/checks/blocks/needbraces.html#Description

Done @romani

Reply:
> `// OK, 'break' is not counted to allow compact view

Updated the comment